src/main/java/racingcar/RacingGameApplication.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import racingcar.car.Car;
+import racingcar.car.CarFactory;
+import racingcar.car.RacingGame;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+import java.util.List;
+
+public class RacingGameApplication {
+
+    public static void main(String[] args) {
+        int numberOfCars = InputView.getNumberOfCars();
+        int tryCount = InputView.getTryCount();
+
+        List<Car> cars = CarFactory.create(numberOfCars);
+        RacingGame racingGame = new RacingGame(cars);
```

Comment:
```suggestion
        Cars cars = Cars.of(numberOfCars);
        RacingGame racingGame = new RacingGame(cars);
```
`Cars` 라는 일급컬렉션을 만들어서 관리해주면 어떨까요?